loky/backend/popen_loky_win32.py

```diff
@@ -58,10 +59,9 @@ def __init__(self, process_obj):
             with open(wfd, 'wb') as to_child:
                 # start process
                 try:
-                    inherit = sys.version_info[:2] < (3, 4)
```

Comment:
Just for the sake of understanding, why do you need `inherit=True` for all version?

Reply:
**Remainder**: 
Whatever the value of `inherit`, we need to pass the write end of the `resource_tracker` to the child process.

* `inherit=True` means that all `inheritable` handles in the parent process will be usable in the child. Therfore, to pass the write end of the `resource_tracker`, we can simply create an inheritable version of it in the parent, and pass its value to the child in `preparation_data`
* `inherit=False` means that no `handle` can be inherited from the parent to its children. Therefore, to communicate the write end of the `resource_tracker` to its child,  one has to open the parent process handle, snag the pipe handle we want,  and duplicate it - **all of this in the child process**. 

One can argue that the second mechanism is cleaner than the first one. Indeed, in the first case, no parent-process inherited file/pipe handle is  initially available in the child process.


**HOWEVER**:

* this mechanism relies on `winapi` functions like `OpenProcess`, that are only available in `python>=3.4`.
* coincidentally, keep in mind that the case where `inherit=False` used to be implemented natively in `python multiprocessing`, as for `python >= 3.4`, `multiprocessing.spawn.popen_spawn_win32.main` was used as child processes starting command. So the code complexity of this case was abstracted away from us.

Now, `loky.spawn.popen_loky_win32.main` is used for all python versions, to take into account `resource_tracker` information passing at the begining of the process. So if I want to use `inherit=False` (thus steal handles etc.) for `python >= 3.4`, the code will have to be written inside loky. And it is going to result in more version branching.

So I decided to rely only on the first case to pass file handles between a parent and a child, to avoid adding more complexity to the code.

Reply:
Thanks for the explanation. I think this is fine as is but could you add a comment stating that when we drop `2.7` support, this should be changed?